mmv1/third_party/terraform/utils/gcp_sweeper_test.go

```diff
@@ -17,6 +17,7 @@ var testResourcePrefixes = []string{
 	"df-",                 // https://github.com/hashicorp/terraform-provider-google/issues/8909
 	"resourcegroup-",      // https://github.com/hashicorp/terraform-provider-google/issues/8924
 	"cluster-",            // https://github.com/hashicorp/terraform-provider-google/issues/8924
+	"k8s-fw-",
```

Comment:
Can you include context on why this is a value we look for?

Reply:
done !